res/css/_spacing.pcss

```diff
@@ -14,8 +14,16 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-/* 1rem :: 10px */
-
+/* SCSS variables representing a range of standard lengths.
+ *
+ * Avoid using these in new code: we cannot adjust their values without causing massive confusion, so they are
+ * effectively equivalent to using hardcoded values.
+ *
+ * In future, we plan to introduce variables named according to their purpose rather than their size. Additionally,
+ * we want switch to custom CSS properties (https://github.com/vector-im/element-web/issues/21656), so we might have
+ * `--spacing-standard` or something. For now, you might as well use hardcoded px values for lengths (except for font
+ * sizes, for which see the `$font-<N>px` variables).
```

Comment:
Yes, this will be done as part of what https://github.com/vector-im/compound-design-tokens outputs.
I've commented on the linked issue to explain a rough path to a better future